src/03_async_await/01_chapter.md

```diff
@@ -48,7 +48,7 @@ into a `'static` future is to bundle the arguments with the call to the
 ```
 
 By moving the argument into the `async` block, we extend its lifetime to match
-that of the `Future` returned from the call to `foo`.
+that of the `Future` returned from the call to `borrow_x`.
```

Comment:
Good catch on the typo here! I think this should actually be the  `good`. We're talking about the future returned from `good()` (which `x` lives as long as). `x` actually lives slightly longer than the future returned by `borrow_x` here.

Reply:
I think you are correct. I was thinking about how long `x` needed to live (as long as `borrow_x` can access it), but we are indeed extended its lifetime to be the same as that of the `Future` returned by `good`.